### PR TITLE
[ADD] stock_picking_invoice_link: stock picking reports as attachment…

### PR DESCRIPTION
**Pull request by catalin-domatix**
Given an invoice with pickings, add its pickings `Picking Operations` pdf reports as attachments to the email created by the `Send by Email` invoice button.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This is something that shouldn't be done as a general thing, as not everyone wants this. I would suggest an option at res.partner level, company level or system-wide parameter for deciding this.

**Comment by catalin-domatix** [comment]
> Hey @catalin-domatix, thank you for your Pull Request.
> 
> It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet.
> You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/cla
> Here is a list of the users:
> 
>     * @catalin-domatix (login unknown in OCA database)
> 
> 
> Appreciation of efforts,
> OCA CLAbot

From what my colleagues told me this issue has been resolved. 

On the other hand, the Travis lint checkers have failed on files which are alien to the files modified on this PR, so having added an option for the user to select if they want to have the picking reports attached to the email, I'd ask if there is something more to do in order for this PR to be accepted.

**Review by alvaro-domatix — APPROVED** [review]

**Review by samuel-domatix — APPROVED** [review]

**Comment by rousseldenis** [comment]
@catalin-domatix Could you please fix lint errors (flake8) ?

**Review by alvaro-domatix — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review.

**Comment by rousseldenis** [comment]
@catalin-domatix Could you rebase ?

**Comment by rousseldenis** [comment]
@catalin-domatix Wrong rebase. Could you fix it ?

**Comment by catalin-domatix** [comment]
@rousseldenis what exactly is wrong with the rebase tho? i genuinely don't know

**Comment by rousseldenis** [comment]
@catalin-domatix Ok, don't see anymore. Seems to be my browser cache. :

**Comment by rousseldenis** [comment]
Are all your commits relevant or do you want to squash some of them ?

**Comment by catalin-domatix** [comment]
i'd say none of them are relevant besides the first and second one but squashing is not trivial since i rebased, there are commits which are not mine in between my own ones, gimme a minute

File: stock_picking_invoice_link/i18n/es.po [hunk 1]
```diff
@@ -101,5 +101,22 @@ msgstr "Transferir"
 msgid "You can not modify an invoiced stock move"
 msgstr "No puede modificar un movimiento de almacén facturado"
 
+<<<<<<< HEAD
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/i18n/es.po:104]: This conflict must be solved

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: stock_picking_invoice_link/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -18,7 +18,9 @@
 
   * Pedro M. Baeza <pedro.baeza@tecnativa.com>
   * David Vidal <david.vidal@tecnativa.com>
+* `Domatix <https://www.domatix.com>`_
 
+  * Catalin Airimitoaie <catalin@domaitx.com>
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/readme/CONTRIBUTORS.rst:23]: Incorrect email address

File: stock_picking_invoice_link/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -1,13 +1,18 @@
 This module adds a link between pickings and invoices as well as on the lines.
-Invoices are generated from sales orders. With this module, you can find back
+Invoices are generated from sales orders. With this module you can find back
```
Comment (pedrobaeza) [inline stock_picking_invoice_link/readme/DESCRIPTION.rst:2]: Why do yo remove this valid comma?

File: stock_picking_invoice_link/i18n/es.po (hunk 1 above)
Comment (rousseldenis) [reply stock_picking_invoice_link/i18n/es.po:104]: That's what I wanted to say. My memory is limited :smile:

**Pull request closed without merging**